public/js/main.js

```diff
@@ -113,6 +113,7 @@
 
 				// NOTE: in version v0.0.1 simple HTML POST will do fine
 				$.post('/tables', object);
+				window.location.href = '/tables';
```

Comment:
Good behaviour fix, but let us do separated PRs for many tasks next time, to have better vision